sagemaker-neo-notebooks/edge/cpp-integration/tutorial.cc

```diff
@@ -0,0 +1,622 @@
+#include <chrono>
+#include <ctime>
+#include <exception>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+// 3rd-party
+#include <nlohmann/json.hpp>
+#include "npy.hpp"
+
+// aws-sdk
+#include <aws/core/Aws.h>
+#include <aws/core/client/AWSError.h>
+#include <aws/iam/IAMClient.h>
+#include <aws/iam/model/AttachRolePolicyRequest.h>
+#include <aws/iam/model/CreateRoleRequest.h>
+#include <aws/iam/model/GetRoleRequest.h>
+#include <aws/iam/model/ListRolesRequest.h>
+#include <aws/s3/S3Client.h>
+#include <aws/s3/model/CreateBucketRequest.h>
+#include <aws/s3/model/GetObjectRequest.h>
+#include <aws/s3/model/ListObjectsRequest.h>
+#include <aws/s3/model/PutObjectRequest.h>
+#include <aws/sagemaker/SageMakerClient.h>
+#include <aws/sagemaker/model/CreateCompilationJobRequest.h>
+#include <aws/sagemaker/model/DescribeCompilationJobRequest.h>
+#include <aws/sagemaker/model/Framework.h>
+#include <aws/sagemaker/model/InputConfig.h>
+
+// internal
+#include <dlr.h>
+
+std::string ROLE_NAME = "windows-demo-test-role";
+
+Aws::S3::S3Client getS3Client() {
+  // for tutorial, we set region to us-west-2
+  std::string region = "us-west-2";
+  Aws::String region_aws_s(region.c_str(), region.size());
```

Comment:
Does the following work? `Aws::String region_aws_s("us-west-2")`

Reply:
yeah, that'll work.